modules/diskq/dqtool.c

```diff
@@ -526,9 +531,29 @@ _assign_validate_options(void)
   return _validate_persist_file_path(persist_file_path);
 }
 
+static void
+_assign_print_help(void)
+{
+  fprintf(stderr, "example:"
+          "   bin/dqtool assign -p var/syslog-ng.persist -n \"afsocket_dd_qfile(stream,localhost:15554)\""
+          "                    /tmp/syslog-ng-dq/syslog-ng-00000.rqf\n\n"
+          "When only a filename is given for diskq, it will be appended to the current working dir.\n"
+          "One bad thing: user need to figure out the correct persist name.\n\n");
+  fprintf(stderr, "How it works?\n"
+          "When you know what will the persist name for the diskq file but you want to manually assign"
```

Comment:
"When you know what the persist name for the diskq file is, and you want to manually..."